src/utils/queryParams.ts

```diff
@@ -0,0 +1,19 @@
+export function formatQueryParams(
```

Comment:
there may be a lib we're already using that offers a util like this, but this is simple and works

Reply:
We can use `qs` for this. 

https://github.com/ljharb/qs#usage

Think we should prefer to rely on tested libs for this kinda stuff.